### PR TITLE
update on models and data sources for Italy, France, UK

### PR DESCRIPTION
**Pull request by Liyubov**
updated data from https://github.com/pcm-dpc/COVID-19 and some corrections for references

### CODE REVIEW

File: coronavirus.md [hunk 1]
```diff
@@ -62,10 +62,51 @@ Coming soon ...
...
+    data/covid-19-indicators-uk.csv: daily counts of tests, confirmed cases, deaths for the whole of the UK and individual countries in the UK (England, Scotland, Wales, Northern Ireland). This is a tidy-data version of covid-19-totals-*.csv combined into one file.
+
 ### France
 
+Data source: https://github.com/cedricguadalupe/FRANCE-COVID-19
+
+
+Data information: 
+Data is updated daily according to the main sources of information: 
+Agence Regionale de Sante;    Santé Publique France : https://www.santepubliquefrance.fr/;   Geodes: https://geodes.santepubliquefrance.fr/#c=indicator&view=map2
```
Comment (rufuspollock) [inline coronavirus.md:103]: Is this the official source? The github is not right?

File: coronavirus.md [hunk 2]
```diff
@@ -62,10 +62,51 @@ Coming soon ...
...
+    data/covid-19-totals-uk.csv: daily counts of tests, confirmed cases, deaths for the whole of the UK
+    data/covid-19-totals-england.csv: daily counts of tests, confirmed cases, deaths for England
+    data/covid-19-totals-northern-ireland.csv: daily counts of tests, confirmed cases, deaths for Northern Ireland
+    data/covid-19-totals-scotland.csv: daily counts of tests, confirmed cases, deaths for Scotland
+    data/covid-19-totals-wales.csv: daily counts of tests, confirmed cases, deaths for Wales
+    data/covid-19-indicators-uk.csv: daily counts of tests, confirmed cases, deaths for the whole of the UK and individual countries in the UK (England, Scotland, Wales, Northern Ireland). This is a tidy-data version of covid-19-totals-*.csv combined into one file.
+
 ### France
 
+Data source: https://github.com/cedricguadalupe/FRANCE-COVID-19
```
Comment (rufuspollock) [inline coronavirus.md:98]: Why two 2 blank lines here?

File: coronavirus.md [hunk 3]
```diff
@@ -62,10 +62,51 @@ Coming soon ...
...
+
+
+Data information:
```
Comment (rufuspollock) [inline coronavirus.md:101]: Blank line after the `:`?

File: coronavirus.md (hunk 1 above)
Comment (Liyubov) [reply coronavirus.md:103]: yes, true, most of the links (not on github) I found here https://github.com/open-covid-19/data I am verifying them now...

**Pull request merged**